feature/feature.go

```diff
@@ -47,6 +47,11 @@ const (
 	// owner: @upxinxin
 	// alpha: v1.8
 	EdgeZone featuregate.Feature = "EdgeZone"
+
+	// NodePrototyping is the feature gate for having prototyping mode on
+	// owner: @bennycortese
+	// alpha: v1.11
+	NodePrototyping featuregate.Feature = "NodePrototyping"
```

Comment:
how does this get enabled by the user? we might be missing an addition in manager.yaml

Reply:
I mentioned my opinion above: https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/3727#issuecomment-1636217139

Reply:
Got it. Would it make sense to still add the environment variable to the yaml template but hardcoding it to `false` for now to make it easier to quickly switch the value for development?

Reply:
Would changing the default value below also allow easily toggling it? Even if so, a placeholder for the feature flag hardcoded to false would be a better reminder that we should update it